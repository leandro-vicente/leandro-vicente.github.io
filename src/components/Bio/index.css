.bio {
  padding-top: 20px;
}

.avatar {
  margin-top: 20px;
  border-radius: 100%;
  height: 200px;
  width: 200px;
}

@media only screen and (max-width: 768px) {
  height: 150px;
  width: 150px;
}