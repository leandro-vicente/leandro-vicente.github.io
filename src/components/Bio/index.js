import React from 'react'
import PropTypes from 'prop-types'

const Bio = (props) => {
  return (
    <React.Fragment>
      <h1>{props.name}</h1>
      <h2>{props.tagline}</h2>
    </React.Fragment>
  )
}

Bio.propTypes = {
  name: PropTypes.string.isRequired,
  tagline: PropTypes.string.isRequired
}

export default Bio