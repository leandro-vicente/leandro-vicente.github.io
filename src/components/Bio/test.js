import React from 'react'
import Bio from './index'
import { shallow } from 'enzyme'

describe('Bio', () => {
  it('renders as expected', () => {
    const bio = shallow(<Bio />)
    expect(bio).toMatchSnapshot()
  })
})