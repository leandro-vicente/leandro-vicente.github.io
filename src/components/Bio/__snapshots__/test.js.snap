// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bio renders as expected 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Bio />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <h1 />,
        <h2 />,
        <SocialMediaList
          iconsAndLinks={
                    Array [
                              Object {
                                "glyphicon": "fa fa-linkedin fa-2x",
                                "name": "linkedin",
                                "url": "https://www.linkedin.com/in/leandrovicentesilva/",
                              },
                              Object {
                                "glyphicon": "fa fa-github fa-2x",
                                "name": "github",
                                "url": "https://github.com/llevvi/",
                              },
                              Object {
                                "glyphicon": "fa fa-youtube-play fa-2x",
                                "name": "youtube",
                                "url": "http://youtube.com/levvitalk",
                              },
                              Object {
                                "glyphicon": "fa fa-instagram fa-2x",
                                "name": "instagram",
                                "url": "http://instagram.com/levvitalk",
                              },
                            ]
          }
/>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "iconsAndLinks": Array [
            Object {
              "glyphicon": "fa fa-linkedin fa-2x",
              "name": "linkedin",
              "url": "https://www.linkedin.com/in/leandrovicentesilva/",
            },
            Object {
              "glyphicon": "fa fa-github fa-2x",
              "name": "github",
              "url": "https://github.com/llevvi/",
            },
            Object {
              "glyphicon": "fa fa-youtube-play fa-2x",
              "name": "youtube",
              "url": "http://youtube.com/levvitalk",
            },
            Object {
              "glyphicon": "fa fa-instagram fa-2x",
              "name": "instagram",
              "url": "http://instagram.com/levvitalk",
            },
          ],
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <h1 />,
          <h2 />,
          <SocialMediaList
            iconsAndLinks={
                        Array [
                                    Object {
                                      "glyphicon": "fa fa-linkedin fa-2x",
                                      "name": "linkedin",
                                      "url": "https://www.linkedin.com/in/leandrovicentesilva/",
                                    },
                                    Object {
                                      "glyphicon": "fa fa-github fa-2x",
                                      "name": "github",
                                      "url": "https://github.com/llevvi/",
                                    },
                                    Object {
                                      "glyphicon": "fa fa-youtube-play fa-2x",
                                      "name": "youtube",
                                      "url": "http://youtube.com/levvitalk",
                                    },
                                    Object {
                                      "glyphicon": "fa fa-instagram fa-2x",
                                      "name": "instagram",
                                      "url": "http://instagram.com/levvitalk",
                                    },
                                  ]
            }
/>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "iconsAndLinks": Array [
              Object {
                "glyphicon": "fa fa-linkedin fa-2x",
                "name": "linkedin",
                "url": "https://www.linkedin.com/in/leandrovicentesilva/",
              },
              Object {
                "glyphicon": "fa fa-github fa-2x",
                "name": "github",
                "url": "https://github.com/llevvi/",
              },
              Object {
                "glyphicon": "fa fa-youtube-play fa-2x",
                "name": "youtube",
                "url": "http://youtube.com/levvitalk",
              },
              Object {
                "glyphicon": "fa fa-instagram fa-2x",
                "name": "instagram",
                "url": "http://instagram.com/levvitalk",
              },
            ],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
